src/site/asn1.rst

```diff
@@ -0,0 +1,75 @@
+Assignment #1: TBD
+==================
+
+* **Worth**: 10%
+* **DUE**: TBD at 11:55pm; submitted on MOODLE.
+
+
+What to submit to Moodle
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Submit your work on Moodle. 
+
+* Your ``.java`` files.
+
+* Make sure your **NAME** and **STUDENT NUMBER** appear in a comment at the top of the program.
+
+
+**VERIFY THAT YOUR SUBMISSION TO MOODLE WORKED!**
+**IF YOU SUBMIT INCORRECTLY, YOU WILL GET A 0**
+
+
+Some hints
+^^^^^^^^^^^
+
+* Work on one function at a time. 
+* Get each function *working perfectly* before you go on to the next one. 
+* *Test* each function as you write it. 
+    * This is a really nice thing about programming: you can call your functions and see what result gets returned. Does it seem correct?
```

Comment:
Or even test the function _before you write it_. What problem is the function going to solve?